### ISSUE TITLE
gen_langs.py: scripts source all empty

### ISSUE THREAD

**Issue by powerumble** [pndurette/gTTS#350]
### Prerequisites

- [x] Did you make sure a similar [issue](https://github.com/pndurette/gTTS/) didn't exist?
- [x]  Did you update gTTS to the latest? (pip install --upgrade gTTS)

### Proposed Behaviour

generate the latest language dictionary

`python gTTS/scripts/gen_langs.py gTTS/gtts/langs.py`

### Context
Generating language dictionary fails:
```
Traceback (most recent call last):
  File "/tmp/gTTS/scripts/gen_langs.py", line 119, in <module>
    langs = _fetch_langs()
  File "/tmp/gTTS/scripts/gen_langs.py", line 89, in _fetch_langs
    all_langs_raw = data_by_key["ds:3"]
KeyError: 'ds:3'
```
**s.text** is always empty (**scripts[0].text = ""**, **scripts[1].text = ""**, **scripts[2].text = ""**, ...)

According to the beautifulsoup4 documentation _script_, _style_, and _template_ tags are not considered to be ‘text’, since those tags are not part of the human-visible content of the page.
### Patch
```
--- gen_langs.py	2022-05-04 20:27:32.683151920 +0200
+++ gen_langs_new.py	2022-05-04 20:28:39.108060879 +0200
@@ -75,7 +75,7 @@
     soup = BeautifulSoup(page.content, 'html.parser')
 
     scripts = soup.find_all(name='script', string=re.compile(r"^AF_initDataCallback"))
-    scripts = [s.text for s in scripts]
+    scripts = [s.string for s in scripts]
 
     data_by_key = _get_data_by_key(scripts)
```
### Environment
- Python 3.9
- Beautifulsoup4 (4.9.3)
- Linux 5.11.0-49-generic

**Comment by pndurette** [comment]
Hi @powerumble, thanks! I wonder what changed, since it used to work, but thanks for the research and patch! Will take a look.

**Comment by pndurette** [comment]
Fixed in #369
